TTS/tts/models/vits.py

```diff
@@ -1449,6 +1517,11 @@ def load_checkpoint(
         # TODO: consider baking the speaker encoder into the model and call it from there.
         # as it is probably easier for model distribution.
         state["model"] = {k: v for k, v in state["model"].items() if "speaker_encoder" not in k}
+
+        if self.args.TTS_part_sample_rate is not None and eval:
```

Comment:
Why don't you just set it None discarding the if statement

Reply:
Because it is useful in "not eval" mode.  And it needs to be before the checkpoint is loaded otherwise it will break the load_checkpoint (torchaudio.transforms.Resample is not saved in the model state dict). Maybe change it to "if eval:" but it will motivate someone to move it to the end and it will generate an error in inference time.